modules/consentManagementTcf.ts

```diff
@@ -22,11 +24,55 @@ const cmpCallMap = {
   'iab': lookupIabConsent,
 };
 
+export type TCFConsentData = {
```

Comment:
```suggestion
/**
 * @see https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework
 * @see https://github.com/InteractiveAdvertisingBureau/iabtcf-es/tree/master/modules/core#iabtcfcore
 */
export type TCFConsentData = {
```

At some point we could consider depending on the types here: https://github.com/InteractiveAdvertisingBureau/iabtcf-es/tree/master/modules/core#iabtcfcore